provisioning/roles/base/tasks/main.yml

```diff
@@ -15,8 +15,8 @@
   lineinfile:
     dest="/etc/resolvconf/resolv.conf.d/base"
     line="nameserver 8.8.8.8"
-  sudo: yes
+  become: yes
 
 - name: Update nameserver list
   command: "resolvconf -u"
-  sudo: yes
+  become: yes
```

Comment:
what are the implications of those changes?

Reply:
`sudo` is mostly used prior to Ansible 1.9 and [deprecated](https://docs.ansible.com/ansible/latest/user_guide/become.html#for-those-from-pre-1-9-sudo-and-su-still-work). We use Ansible 2.5 at the moment. Also, `googlecompute` builder of Packer doesn't recognize and ignore `sudo` which results in failed Ansible tasks.